app/presenters/step_by_step_page_presenter.rb

```diff
@@ -0,0 +1,23 @@
+class StepByStepPagePresenter
+  include TimeOptionsHelper
+
+  def initialize(step_by_step_page)
+    @step_by_step_page = step_by_step_page
+  end
+
+  def summary_metadata
+    last_saved_time = format_full_date_and_time(@step_by_step_page.updated_at)
+    last_saved = "#{last_saved_time} by #{@step_by_step_page.assigned_to}" if @step_by_step_page.assigned_to.present?
+    items = {
+      "Status": @step_by_step_page.status[:text],
+      "Last saved": last_saved || last_saved_time,
+      "Created": format_full_date_and_time(@step_by_step_page.created_at),
+    }
+    if @step_by_step_page.links_checked?
+      items = items.merge(
```

Comment:
Warning: if any other key/value pairs need to be added, this will need to change to a `merge!`. That's because `merge` returns a new hash rather than mutating the original object, so any new elements added after this will be added to the original `items`.

Another option is to do `items["Links checked"] = format_full_date_and_time(@step_by_step_page.links_last_checked_date)` as that also mutates the original object.

Reply:
Hmm this would still work, as I reassign `items = items.merge`, so an additional key/value pair would just be an additional `items = items.merge`.

In the alternative suggestion, I think it would need to be `items["Links checked".to_sym] =`, which doesn't read very nicely. But I think `items.merge!` would be cleaner 👍